### PR TITLE
fix(flow): cancel pending task cause "Some tasks is not found"

### PR DESCRIPTION
**Pull request by krihy**
#### What type of PR is this?

type-bug
module-flow
#### What this PR does / why we need it:
flowableAdaptor#getFlowableElementByType will return `CALL_BACK` task can  cause` Verify.verify(tasks.size() == elements.size())`  exception, so  filter `CALL_BACK` user task in  getFlowableElementByType result
#### Which issue(s) this PR fixes:

cancel pending task which is timer execute cause "Some tasks is not found"

#### Special notes for your reviewer:

test passed
#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/model/CallBackUserTaskConstants.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.flow.task.model;
+
+/**
+ * @author yaobin
+ * @date 2024-04-22
+ * @since 4.2.4
+ */
+public class CallBackUserTaskConstants {
```
Comment (yhilmare) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/model/CallBackUserTaskConstants.java:23]: RuntimeTaskConstants may meet your needs

Comment (krihy) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/model/CallBackUserTaskConstants.java:23]: looks more better

Comment (krihy) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/model/CallBackUserTaskConstants.java]: done

**Review by yhilmare — APPROVED** [review]
lgtm

**Pull request merged**